.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: kind/bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**OpenShift (please complete the following information):**
+ - Distribution: [e.g OCP]
+ - Client Version [e.g. `Client Version: 4.6.8`]
```

Comment:
Operator csv version?
Commit hash?

Reply:
agree the client version is less interesting than the operator version. On the other hand that doesn't make sense until we have releases.

Reply:
that's actually why i let it out @NautiluX 
happy to adjust it as soon as we have a release

Reply:
Is any user/customer running route monitor operator in non-test environment already?
If not, before deploying rmo on first production cluster by then we should have first release.
I agree with adding the operator version field. Eventually it will be useful.

Reply:
I think we currently have the csv version in the package manifest, so we can use it pre-production